sdk/state/open.go

```diff
@@ -161,6 +161,16 @@ func (c *Channel) ConfirmOpen(m OpenAgreement) (open OpenAgreement, authorized b
 		if err != nil {
 			return
 		}
+
+		// If an agreement ever surpasses 2 signatures per tx, error.
+		if len(m.DeclarationSignatures) > 2 || len(m.CloseSignatures) > 2 || len(m.FormationSignatures) > 2 {
+			authorized = false
```

Comment:
arguably we could return `authorized` as true here since it is fully signed 🤔 . Could come in use for whatever code calls this?

Reply:
Nah, I think you should stick with false. It might be authorized, but if you submitted it with extra signatures, the tx would fail.